static/js/context-menu.js

```diff
@@ -513,6 +520,7 @@ var ContextMenu = new (class ContextMenu extends ContextMenuBase {
               href: `/${tree}/source/${jumpref.jumps.idl}`,
               icon: "export-alt",
               section: "jumps",
+              confidence: confidence,
```

Comment:
nit: like in rust [you can just do "confidence" here rather than "confidence: confidence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions) here and below.  (I think if we adopt "prettier" for formatting it probably would do this automatically too?)